### ISSUE TITLE
Papules with white center

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9911]
**Preferred term label:**
Papules with white center

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Clavus durum 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

is this distinct from [Pustule HP:0200039](https://hpo.jax.org/app/browse/term/HP:0200039) ?
If so, please provide additional context

**Comment by MickeySegal** [comment]
https://www.ncbi.nlm.nih.gov/books/NBK470374/ "Corns usually present as flesh-colored dry, hard, rough papules with a whitish center (called the core). "

**Comment by pnrobinson** [comment]
So I think the phenotype might better simply be called Heloma (one of the synonyms -- I think it might be hard to grok the word Corn in isolation), because there are other types of papules with a white center?

**Comment by MickeySegal** [comment]
The terminology we used was not clear.  I've changed our label to "Focal hyperkeratosis with a whitish center (corn; heloma) that is uncomfortable"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000789
- label: Heloma
- parents: [Localized skin lesion (HP:0011355)](https://hpo.jax.org/app/browse/term/HP:0011355)
- pmid: PMID:29262147
- def: A corn, also known as a clavus, heloma, or focal intractable plantar hyperkeratosis, is a type of callosity. Corns are uncomfortable, thickened skin lesions that result from repeated mechanical trauma due to friction or pressure forces.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9911
- synonyms: Focal intractable plantar hyperkeratosis(NONE)|Clavus(NONE)|Corn(NONE)
